python/pyspark/ml/clustering.py

```diff
@@ -95,15 +95,10 @@ class GaussianMixture(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIte
     >>> weights = model.weights
     >>> len(weights)
     3
-    >>> model.gaussiansDF.show()
-    +--------------------+--------------------+
-    |                mean|                 cov|
-    +--------------------+--------------------+
-    |[0.82500000140229...|0.005625000000006...|
-    |[-0.4777098016092...|0.167969502720916...|
-    |[-0.4472625243352...|0.167304119758233...|
-    +--------------------+--------------------+
-    ...
+    >>> model.gaussiansDF.select("mean").head()
```

Comment:
Using fewer digits can not solve the unstable test issue, since the output of ```gaussiansDF.show()``` is pure string, we must match them exactly. Actually ```gaussiansDF.show()``` only shows part of the whole DataFrame, others are omitted, so the original test does not check for all cells of ```gaussiansDF```.